fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -623,10 +623,8 @@ public void execute(TUniqueId queryId) throws Exception {
             context.getState().setError(ErrorCode.ERR_UNKNOWN_ERROR, e.getMessage());
             throw e;
         } catch (UserException e) {
-            LOG.warn("", e);
             // analysis exception only print message, not print the stack
-            LOG.warn("execute Exception. {}, {}", context.getQueryIdentifier(),
-                                    e.getMessage());
+            LOG.warn("execute Exception. {}", context.getQueryIdentifier(), e);
```

Comment:
LOG.warn("execute Exception. {},error={}", context.getQueryIdentifier(), e.getMessage());

Reply:
why delete the execption message ?